physionet-django/user/forms.py

```diff
@@ -308,7 +308,7 @@ class Meta:
         help_texts = {
             'training_course_name':"The name of the human subjects training course you took. e.g. 'CITI Data or Specimens Only Research Course'",
             'training_completion_date': 'The date on which you finished your human subjects training course. Must match the date in your training completion report.',
-            'training_completion_report':"A pdf of the training completion report from your training program. The CITI completion report lists all modules completed, with dates and scores. Do NOT upload the completion certificate.",
+            'training_completion_report':"Upload the completion report from the CITI \"Data or Specimens Only Research\" training program (PDF or image file). The completion report lists all modules completed, with dates and scores. Do NOT upload the completion certificate. If you would like to submit multiple pages, please combine them into a single pdf file.",
```

Comment:
The text is truncated in the description:

![Screen Shot 2019-06-18 at 09 58 12](https://user-images.githubusercontent.com/822601/59690606-a7153500-91af-11e9-9b4a-490031351e8a.png)

Reply:
It doesn't work in triple quotes, and in first single then double. So it will have to go double then single.